.gitignore

```diff
@@ -1,5 +1,4 @@
 test/output
-test/e2e/output
```

Comment:
I think this should stay, because this is still the default location

Reply:
It's not anymore. `e2e` folder will be now created under `test/output`